acceptance/api/v1/application_export_test.go

```diff
@@ -80,6 +81,35 @@ var _ = Describe("AppPart Endpoint", LApplication, func() {
 `, app, domain, domain)))
 	})
 
+	It("retrieves the named application part, manifest", func() {
+		response, err := env.Curl("GET", fmt.Sprintf("%s%s/namespaces/%s/applications/%s/part/manifest",
+			serverURL, v1.Root, namespace, app), strings.NewReader(""))
+		Expect(err).ToNot(HaveOccurred())
+		Expect(response).ToNot(BeNil())
+		defer response.Body.Close()
+
+		bodyBytes, err := io.ReadAll(response.Body)
+		Expect(err).ToNot(HaveOccurred())
+		Expect(response.StatusCode).To(Equal(http.StatusOK), string(bodyBytes))
+
+		expecting := fmt.Sprintf(`name: %s
+configuration:
+  instances: 1
+  routes:
+  - %s
+  appchart: standard
+origin:
+  container: splatform/sample-app
+  git:
+    url: ""
```

Comment:
Probably this is something "old" but I would not expect to get the `git` field returned if empty. In this case the `origin` is `container` and the Git part is not needed.

Reply:
Might be something with the serialization annotations (around `omitempty`).

Reply:
Managed to reduce the data with commit 6df12eacfbeafad4baaaa44f9d8266c38e711a10.
The `git` does not fully go away :(

Reply:
I had a look again and the issue about the `git` origin is that we are always instanciating it, so that's why the `omitempty` is not working: https://github.com/epinio/epinio/blob/main/internal/application/origin.go#L30-L32

We should create this Origin only for a Git one: https://github.com/epinio/epinio/blob/main/internal/application/origin.go#L79